software/pinepods.yml

```
@@ -0,0 +1,13 @@
+name: "Pinepods"
+website_url: "https://www.pinepods.online/"
+source_code_url: "https://github.com/madeofpendletonwool/PinePods"
+description: "PinePods is a Rust based podcast management system that manages podcasts with multi-user support and relies on a central database with clients to connect to it. It's browser based and your podcasts and settings follow you from device to device due to everything being stored on the server. It works on mobile devices and can also sync with a Nextcloud server so you can use external apps Like Antennapod as well!"
```

> description of what the software does, shorter than 250 characters, sentence case

A possible alternative description below, but feel free to change it however you want to make it get it under 250 characters.

```suggestion
description: "A podcast management system with multi-user support which relies on a central database with clients to connect to it."
```